src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -246,6 +252,8 @@ def _isBlockTierAllowed(self, blockName):
         be handled by this component, thus block-level rule creation as well
         :return: True if the component can proceed with this block, False otherwise
         """
+        if not self.listTiersToInject:
```

Comment:
This will work only if `self.listTiersToInject` is set. Meaning that we should guarantee a default value of `[]` in the constructor. Otherwise it will raise a `NameError` exception.

Reply:
It's set in the init method

Reply:
Default value is already set in the WMAgent configuration template:
https://github.com/dmwm/WMCore/blob/master/etc/WMAgentConfig.py#L373

if someone screws up that variable (either by removing it from the config or something else), the component will crash, as any of the other component attributes.

This attribute will no longer be necessary when we completely migrate to Rucio. Having said that, I think changing this PR again isn't worthwhile.